src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -285,6 +291,14 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
             from helm.benchmark.window_services.lit_gpt_window_service import LitGPTWindowServce
 
             window_service = LitGPTWindowServce(service)
+        elif model_name == "HuggingFaceM4/idefics-9b":
+            window_service = IDEFICS9bWindowService(service)
```

Comment:
You can just do `window_service = HuggingFaceWindowService(None, HuggingFaceHubModelConfig.from_string("HuggingFaceM4/idefics-9b"))`